recipes/hyperscan/all/conanfile.py

```diff
@@ -1,9 +1,13 @@
-from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeToolchain
+from conan.tools.files import apply_conandata_patches, get, copy, rmdir
+from conan.tools.scm import Version
+from conan.tools.build import check_min_cppstd
 
 import os
 
-required_conan_version = ">=1.43.0"
+required_conan_version = ">=1.53.0"
```

Comment:
```suggestion
required_conan_version = ">=1.51.1"
```

Because you need self.info.settings.get_safe: https://docs.conan.io/en/latest/changelog.html#id63